src/PowerShellEditorServices/Services/DebugAdapter/Debugging/VariableDetails.cs

```diff
@@ -290,7 +290,7 @@ private VariableDetails[] GetChildren(object obj, ILogger logger)
                         childVariables.AddRange(
                             psObject
                                 .Properties
-                                .Where(p => p.MemberType == PSMemberTypes.NoteProperty)
+                                .Where(p => p.MemberType == PSMemberTypes.NoteProperty || p.MemberType == PSMemberTypes.Property)
```

Comment:
Might be worth making this `(PSMemberTypes.Properties & p.MemberType) is not 0`.  Though, depending on how we eval `ScriptProperty` might blow up.

Reply:
Yeah absolutely, so `Property` only actually includes CLR bound properties (those defined on the .NET class itself).  There are a few other types of ETS properties that this check will miss like `AliasProperty`, `CodeProperty`, and `ScriptProperty` (the last of which will explode if we try to eval off the pipeline thread so maybe skip it)

`Properties` is a bit mask that includes all of the enum values for every type of property.  So it's equivalent to `AliasProperty | NoteProperty | ScriptProperty | Etc`.

Reply:
I'm deferring to Patrick on this and suggest we do that! 😄 Then we can merge and release!